Classes/Domain/Http/AbTestingCookieComponent.php

```diff
@@ -47,7 +47,7 @@ public function handle(ComponentContext $componentContext)
         $response = $componentContext->getHttpResponse();
 
         $cookieParams = $request->getCookieParams();
-        if ($cookieParams && !array_key_exists($this->cookieSettings['name'], $cookieParams)) {
+        if (array_key_exists($this->cookieSettings['name'], $cookieParams) === false) {
```

Comment:
since array_key_exists only returns booleans, we still can use !array_key_exists

Reply:
True that :)